\documentclass[12pt]{article}
\usepackage[margin = 2.4cm]{geometry} % For margins of 3cm
\usepackage{gensymb} % For some symbols
\usepackage{amsfonts, amssymb, amsmath} % All three for maths symbols
\usepackage[export]{adjustbox} % For figure frames
\setlength{\parskip}{6pt} % To make nice looking paragraph spacing
\usepackage[export]{adjustbox} % For figure frames
\usepackage[numbers, super, sort&compress]{natbib} % bibliographies
\usepackage{setspace} % Allows double spacing
\usepackage{pdfpages} % Allows including PDFs
\usepackage{longtable} % Allows longtables
\usepackage{lscape} % To allow some pages to be in landscape
\usepackage{float, subfloat} % For H flag and subfloats
\usepackage{caption, subcaption} % for ABCD Subcaptions
\usepackage[framemethod=tikz]{mdframed} % Allows coloured boxes

% Longtable settings
\setlength\LTleft{0pt} % flush left
\setlength\LTright{0pt} % flush right
\setlength{\LTcapwidth}{8in}

% mdframed settings
\newmdenv[innerlinewidth=1.5pt, roundcorner=4pt,linecolor=mycolor,innerleftmargin=20pt,
innerrightmargin=20pt,innertopmargin=20pt,innerbottommargin=20pt,leftmargin =30, rightmargin=30]{mybox}
\definecolor{mycolor}{rgb}{0.122, 0.435, 0.698}


\doublespacing % Makes all lines (not figure legends) double spaced

\title{\vspace{-2cm} Transcriptome and epigenome profile of B cells in patients with Transient Hypogammaglobulinemia of Infancy}
\date{}
\author{Brittany Howell}

\newcommand{\CDH}{CD4$^+$ } %Thelper
\newcommand{\CDR}{CD4$^+$CD25$^+$ } %Treg
\newcommand{\IFNG}{IFN-$\gamma$ }
\newcommand{\NFKB}{NF-$\kappa$B }
\newcommand{\naive}{na\"{i}ve }
\newcommand{\Naive}{Na\"{i}ve }

\newcommand{\Th}{T$_\textrm{H}$ }
\newcommand{\ThO}{T$_\textrm{H}1$} 
\newcommand{\ThT}{T$_\textrm{H}2$ }
\newcommand{\Tc}{T$_\textrm{C}$ }

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%%%%  Word count: 2500  %%%%%%%%%%  
%%%%%  Inc. all legends & titles  %%%%%  
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%


\begin{document}
	\maketitle
	
	\paragraph{Hypothesis}
	~\\
	Patients with Transient Hypogammaglobulinemia of Infancy (THI) will exhibit delayed loss of methylation in B cell lymphopoiesis genes, resulting in a deficiency of mature B cell subpopulations.
	
	\section{Background}
		
		Antibodies are a vital component of the adaptive immune system \citep{Simon15}. 
		The production of antibodies occurs when \naive B cells are activated in response to foreign antigens (figure \ref{fig:BCellDevelopment}).
		During gestation, maternal antibodies (mAbs) are passed to the foetus through the placenta \citep{Hasselquist09}.
		At birth, B cells do not complete maturation and so are unable to produce antibodies \citep{Martin10,Rechavi15}.
		Hence in normal infants, antibody concentration decreases until reaching its lowest level at 4 to 6 months of age, a point referred to as physiologic agammaglobulinemia. 
		As the immune system develops, antibody production begins to substantiate normal serum antibody levels \citep{Dressler89}.
		Transient Hypogammaglobulinemia of infancy (THI) is a disorder where regular hypogammaglobulinemia is prolonged or exacerbated then spontaneously alleviated. \citep{Dressler89,AlHerz14,Gitlin56,AlHerz11,Rosen66,McGeady87,Stiemh80, Dalal98,Tiller78,Buckley83}.
		The mechanism causing THI is unknown \citep{AlHerz14}. 

			
		\paragraph{Cause of THI} 
			~\\
			Studies investigating THI have found that levels of circulating B cells are normal and subpopulations of B cells are intact \citep{Tiller78,Stiemh80,Siegel81,Buckley83,Fiorilli86,Dressler89}.
			Furthermore, upon antigenic challenge, most patients can produce a normal antibody response \citep{Tiller78,Stiemh80,Buckley83,McGeady87,Dressler89,Dalal98}.
			In regards to a genetic inheritance, THI was initially thought to be hereditary \citep{Willenbockel60, Soothill68}, however proceeding studies have not shown supporting evidence \citep{Tiller78,Fiorilli86,Ovadia14}.
			With apparently normal B cell populations and no genetic basis, the cause of THI has been speculated extensively, but no proposed mechanism has been supported by replicated evidence \citep{Fudenberg64,Rosen66,Nathenson71,Willenbockel60,Soothill68,Tiller78,Fiorilli86,Ovadia14,Siegel81,McGeady87}.
			The most agreed upon cause in the literature is that there is some kind of delay in B cell maturation or activation \citep{McGeady87,Stiemh80,Walker94,Rosen84}. 
			
		\paragraph{B cell development}
			~\\
			Antibody deficiency can result from either lack of production, or aberrant degradation \citep{Fiorilli86}. 
			All antibodies are produced by mature B cells, developed from haematopoietic stem cells in a pathway shown in figure \ref{fig:BCellDevelopment}.
			First, the hematopoietic stem cells differentiate into common lymphoid progenitors, which commit to the B cell lineage \citep{Cooper15}.
			B cells gradually rearrange their immunoglobulin genes and differentiate into mature \naive B cells which leave the bone marrow to enter the periphery. 
			Resting naive B cells transit through lymph nodes where they encounter specific antigen, activating them and inducing the germinal centre reaction. 
			Further rearrangement of immunoglobulin genes occurs followed by rapid proliferation and differentiation into plasma and memory cells. 
			Plasma cells produce the antibodies required for humoral immunity. 
			The isotype (IgM/G/A/E) of the antibodies is determined by the environmental signals present at the activated B cell stage \citep{Pieper13}. 
			Failure to proceed at any stage of B cell development can result in a deficiency of plasma cells and thence antibody deficiency.
			
			
			\begin{figure}[tb]
				\centering
				\includegraphics[width=\linewidth]{../Figures/Bcelldevelopment.png}
				\caption{B cell development from hematopoietic stem cell to memory and plasma cells. Phases are shown as antigen independent or dependent and location (bone marrow or periphery) is indicated. IGM/G/A/E indicates option of antibody isotype. BCR: B cell receptor.}
				\label{fig:BCellDevelopment}
			\end{figure}
			
			B lymphocyte differentiation is tightly regulated by transcription factors (TFs) functioning in a complex network of auto-regulation, cross regulation, and positive and negative feedback loops (figure \ref{fig:TFBcell}) \citep{Choukrallah14,Polansky08,Oakes16,Zan15,Li13,Mercer11,Kulis15,McManus11}.
			TFs prominent in early stages of B cell specification and commitment include Pax5, E2A, Ebf1, Ikaros, PU.1, and FoxO1. 
			They influence B cell development by promoting changes in chromatin remodelling, facilitating DNA methylation or demethylation and interacting with other factors \citep{Gao09,Maier04,Walter08,Decker09,Lin10,McManus11,Treiber10,Zandi00}. 
			Loss of function studies involving the aforementioned TFs show some defect in B cell lineage commitment, resulting in a loss of a B cell subset or the entire B cell lineage \citep{Choukrallah14}.

			
			\begin{figure}
				\centering
				\includegraphics[width = 0.4\linewidth]{../Figures/BcellTF.png}
				\caption{Schematic representation of interactions between early B cell transcription factors. Adapted from \citet{Choukrallah14}.}
				\label{fig:TFBcell}
			\end{figure}
		
		
			\paragraph{Flow Cytometry}
			~\\
			Previous investigations into THI have claimed that subpopulations of B cells are the same in THI patients as controls \citep{Tiller78,Stiemh80,Siegel81,Buckley83,Fiorilli86,Dressler89}. 
			The claim is supported by rosette-formation and single or two-colour membrane immunofluorescence studies. 
			The immunofluorescence studies distinguished only between mature and immature B cells, using the observed combinations of surface IgM and IgG. 
			Analysis techniques have improved immensely since the most recent experimental THI study took place.
			Flow cytometry (FACS) in particular has developed extensively, allowing the measurement of an increasing number of parameters per cell \citep{Saeys16}.
			18-parameter FACS is now routinely used, 30-parameter flow cytometers are becoming commercially available, and 50-parameter FACS is predicted to be available soon \citep{Saeys16}.
			Due to the much larger number of parameters measurable per cell now compared to when THI was last studied, we have a much greater capacity to study the distribution of B cell subtypes than has been previously performed.
			
			
		\paragraph{Lineage commitment}
		~\\
			Epigenetic modifications act in concert with TFs to confer the phenotype of many cell subsets of the immune system \citep{Lara14,Zan15,Mercer11}. 
			A prominent example is the activation and differentiation of the many T cell subsets \citep{Begin14, Zeng13}. 
			Confirmation of the T regulatory cell (Treg) lineage relies on stable expression of FOXP3.
			However, FOXP3 expression itself is not sufficient to establish and maintain the Treg phenotype.
			In fact, FOXP3 expression occurs in all T cell subsets upon activation \citep{Ohkura13,Polansky08}.
			The distinguishing factor between Tregs and other subsets is the methylation status of the FOXP3 gene, otherwise known as the Treg-specific demethylated region (TSDR) \cite{Shimazu16}.
			When the T cell receptor is engaged in Treg progenitors, demethylation occurs at the TSDR. 
			FOXP3 can then bind to its own gene stabilising expression and hence reinforcing commitment to the Treg lineage.
			Contrastingly, in other subsets the TSDR remains methylated and so FOXP3 expression is transient \citep{Ohkura13}.
			Hence methylation status of key genes can permit lineage commitment. 
			
			Aberrant epigenetics have recently been implicated as the cause of common variable immunodeficiency (CVID), a disease similar to THI \citep{Tallmadge15}.
			CVID is a late-onset primary immunodeficiency characterised by dysfunction or loss of B lymphocytes and decreased immunoglobulin production \citep{Cunningham-Rundles12}.
			RNA-Seq analysis identified 103 genes which were differentially expressed between healthy controls and CVID patients \citep{Tallmadge15}.
			The most severely down-regulated gene was the transcription factor Pax5.
			Epigenome analysis revealed that the Pax5 enhancer was hypermethylated.
			Pax5 is essential to commit a cell to the B cell identity through activation of 170 B cell specific genes and repression of at least 110 lineage inappropriate genes \citep{Schebesta07,Delogu06,Roessler07}.
			\citet{Tallmadge15} hypothesised that the methylation of the Pax5 enhancer was silencing the gene, resulting in a decline of B lymphopoiesis in the bone marrow, followed by the depletion of B cells characteristic of CVID.
			
			It is possible that the prolonged antibody deficiency exhibited in THI patients is caused by a similar mechanism of epigenetic silencing.
			DNA methylation was the first epigenetic mechanism recognised, and is the most commonly studied \citep{Begin14}.
			Hence studying methylation of THI samples is a logical first step to understanding the cause of THI.
		
	\section{Summary and experimental aims}

		\begin{mybox}
			
			\begin{itemize}

				\item THI is a self-limited disorder characterised by prolonged deficient levels of serum antibody IgG which gradually subside with no obvious stimulus \citep{Tiller78,Soothill68,Siegel81,McGeady87,Dressler89,Kowalczyk97,Dalal98,AlHerz14,Gitlin56,AlHerz11,Rosen66,Stiemh80,Buckley83}.
				\item The lack of evidence supporting a genetic basis suggests that the cause of THI is not within the genome \citep{Tiller78,Fiorilli86,Ovadia14}.
				\item \citet{Tallmadge15} showed that in CVID, aberrant methylation of key B cell maturation region, Pax5, was the cause of late onset dysfunctional B lymphopoiesis.
				\item It is possible that the limited IgG production exhibited in THI is caused by delayed B cell maturation as a result of inappropriate methylation.
				
			\end{itemize}

		\end{mybox}

			
		
			\paragraph{Aim 1:} Using FACS, describe the B cell subpopulations in THI patients and normal individuals throughout early development.
			
			\paragraph{Aim 2:} Using whole genome bisulphite sequencing, identify regions of the genome which are differentially methylated in THI samples and controls.
			
			\paragraph{Aim 3:} Using RNA-Seq, identify regions which are differentially expressed in THI samples and controls.

	\section{Data collection and management}
	
		\subsection{Collecting samples}
			
			Members from the Pediatric Department at Wolfson Medical centre have kindly agreed to provide whole blood and tonsil samples they have collected from 26 THI patients (table \ref{table:samples}).
			Further peripheral blood, bone marrow and tonsil samples will be obtained from the Women's and Children's Hospital Immunodeficiency clinic. 
			Blood will be taken from THI patients between initial presentation and normalisation of IgG levels.
			Bone marrow will be taken first upon initial presentation, then in six monthly intervals following, until normalisation of IgG levels.
			Control samples will come from donations by healthy subjects. 
			A full clinical record will be kept for every control and experimental sample collected to allow the best possible matching of THI patients with healthy controls. 
		
		\subsection{Matching controls} 
		
			To determine B cell maturation changes caused by THI, it is imperative to compare samples that are at the closest possible environmental stage.
			Maturation of B cells integrates numerous internal and environmental signals, so controlling for developmental stage has significant challenges. 
			Where possible, THI samples will be fully matched to control samples. 
			The most important criteria to match will be age, as it is the most prominent indicator of immune system development. 
			Secondly, the method of both birth (vaginal or caesarian) and feeding (bottle or breastfed) will be considered, as both have a large influence over the immune system \citep{Jakobsson14,Cho13,Brandtzaeg03,Rogier14,Gomez14}.
			Further factors such as gender and ethnicity will be matched if possible. 
			Finally the diet of patients will be standardised to reduce the environmental effect on B cell maturation.
			
			While completely matching samples and controls is preferable, there are limitations in the scope of the matching. 
			In the samples provided, there are already gaps in the clinical history of samples (table \ref{table:samples}). 
			Without information such as ethnicity or mode of birth, it is impossible to match a sample to an appropriate control. 
			Additionally, the control samples will be taken from participants' donations which may not match the clinical history of any THI patient.
			Furthermore, there are a myriad of external factors that will affect the dynamic nature of the immune system, so even with stringent matching, the fact that not all samples are perfectly matched will be considered.
		 
	
	\section{Aim 1:}
	
		\paragraph{Hypothesis} THI samples will be deficient in mature B cell subsets.
		
		\subsection{Proposed experiment}
		
				To analyse the distribution of B cell subpopulations, we will segregate the B cells using flow cytometry as in \citet{Kulis15} and \citeauthor{Oakes16} (\citeyear{Oakes16}) \citep{Oakes16}.
				B cell populations obtained will include progenitor, pre-BI, pre-BII, immature and plasma cells from bone marrow, \naive and memory B cells from peripheral blood and plasma cells and germinal centre and \naive B cells from tonsils. 
				FACS will be used for all three tissues to sort populations using the surface markers in figure \ref{fig:BCellSorting}.
				
				\begin{figure}[tb]
					\centering
					\includegraphics[width=.9\linewidth]{../Figures/BcellSorting}
					\caption{Description of the FACS sorting markers used to segregate B cell populations.}
					\label{fig:BCellSorting} 
				\end{figure}

		
		\subsection{Possible outcomes and interpretations}		
		
		
			Segregating samples into B cell subpopulations serves two purposes. 
			Firstly, sorting allows greater confidence in any observed differences.
			It is imperative that any changes observed between THI and controls are explained by THI rather than normal B cell maturation changes.
			After sorting, we can compare THI and control cells that at the same maturation stage. 
			%Therefore the developmental variation between compared cells will be reduced allowing more confidence that observed changes are due to THI.
			While there are always limitations when applying discrete stages to continuous processes, segregating B cells into such specific subtypes significantly reduces the chance of detecting differences which are due to normal development. 
			Secondly, as the cells will be separated based on maturity, we are able to compare the prevalence of each cell subtype. 
			If there is a block in the B cell development pathway, as in \citet{Tallmadge15}, we will see deficiencies of mature B cells in THI samples compared to controls.
			Alternatively, the distribution of B cell subpopulations could be intact, as previously claimed \citep{Tiller78,Stiemh80,Siegel81,Buckley83,Fiorilli86,Dressler89}.
			A lack of deficient subpopulations indicates that the cause is not in the development of the B cell lineage but instead in IgG production by plasma cells. 
			Therefore FACS allows us to compare cells more accurately and can show if the mechanism affects B cell development.
			
	
	\section{Aim 2:}
	
		\paragraph{Hypothesis:} B cells from THI patients will exhibit hypermethylation at key B cell development regions.
		
		\subsection{Proposed experiment}
		
			To investigate global DNA methylation of THI and control samples, we will produce full methylomes of the B cell lineages in figure \ref{fig:BCellSorting}. 
			Whole genome bisulfite sequencing (WGBS) will be used such that we can obtain base-pair resolution of all methylated cytosines within the genome \citep{Kulis15,Oakes16}. 
			To perform the analysis, we will use two sets of biological replicates for each of the samples. 
			Samples will undergo two rounds of bisulfite conversion to ensure a cytosine to thymine conversion rate of over 99\%.
			Treated samples will then be sequenced on an Illumina HiSeq 2000 platform and mapped to the genome using the STAR algorithm (v2.4.2a) \citep{Dobin13}.
			
			We will then use ChIP-seq data from the ENCODE project \citep{ENCODE-Project-Consortium12} to analyse methylation status in the context of transcription factor binding sites.
			The relative enrichment of each TFBS in any differentially methylated regions will be calculated in comparison to background reads. 
			A Fisher's exact test will be used to assign an odds ratio and \textit{P} value to each comparison. 
			Of particular focus will be the genes which are specific to B lymphopoiesis such as those in figure \ref{fig:TFBcell}.

	\begin{figure}[tb]
		\centering
			%					\flushleft
			\includegraphics[width=.65\linewidth]{../Figures/methylomeMap}
			\caption{Methylome map produced by \citet{Kulis15}. }
			\label{fig:MethMap}

		\caption{Cytosine methylation status is shown for six stages of B cell development. Concentric circles are labelled with developmental stage.}
	\end{figure}
			
		
		\subsection{Possible outcomes and interpretations}

			The methylation data will allow us to produce unbiased DNA methylation maps of each of the sorted cell populations as in figure \ref{fig:MethMap}.
			The map clearly shows the gradual change in methylation exhibited by the cell subsets. 
			It is expected that in both THI and control samples in all age groups, global methylation decreases as the B cells mature, as previously observed \citep{Kulis15,Lai13,Kulis12,Shaknovich11}. 
			If THI causes delayed loss of methylation, it would be clearly shown in methylation maps.
			by comparing the same individuals at different ages, we expect that regions which show hypermethylation will gradually lose their methylation and begin to resemble the controls.
			
			The ChIP-Seq data provides a more detailed view of key regions such as transcription factor binding sites (TFBSs).
			We will produce a heatmap such as that in figure \ref{fig:TFBSheatmap} which displays the correlations between TFBSs and differentially methylated regions. 
			In the THI samples, we expect to find that regions related to B cell maturation or IgG production will be hypermethylated compared to background levels. 
			The location of differentially methylated regions (DMR) will indicate any potential causes of THI.
			If a region associated with B cell maturation, such as Pax5, is found to be differentially methylated it indicates that THI could be a result of a developmental block like that seen in CVID \citep{Tallmadge15}.
			Alternatively if there is a DMR associated with gene rearrangement, THI could be caused by a lack of B cell diversity.
			Any DMRs identified will be further investigated with expression analysis.
			
			\begin{figure}[tb]
				\centering
				\includegraphics[width=.8\linewidth]{../Figures/TFbsKulis}
				\caption{Association of differentially methylated regions (DMRs) and transcription factor binding sites (TFBSs). From \citet{Kulis15}.}
				\label{fig:TFBSheatmap}
			\end{figure}
		
	\section{Aim 3:}
	
		\subsection{Hypothesis}
			
			Samples from THI patients will exhibit decreased expression for B cell commitment associated genes compared to controls.
					
		\subsection{Proposed experiment}
			
			To investigate expression changes across the genome of THI samples, we will conduct RNA-Seq analysis on each of the B cell subsets in figure \ref{fig:BCellSorting}. 
			RNA-Seq libraries will be generated using the TruSeq Stranded Total RNA kit (Illumina). 
			Sequenced reads will be aligned to the genome using the STAR algorithm (v2.4.2a)\citep{Dobin13}.
			qPCR analysis of gene expression will then be undertaken. 
			Target gene expression will be presented relative to average expression for the housekeeping genes \textit{GAPDH}, \textit{ACTB} and \textit{HPRT1}.
			
		\subsection{Possible outcomes and interpretations}				
			
			Expression analysis will allow us to  further investigate any DMRs found in methylation analysis as well as find other genes which are differentially expressed. 
			Heatmaps will be generated to display the genes which change expression between THI and control samples.
			
			Primarily, we expect to see differentially expressed regions associated with IgG production. 
			For example, as Rag1/2 controls gene rearrangement a lowered expression would make sense in terms of lowered antibody production \citep{Choukrallah14}.
			
			Furthermore, we expect genes which show hypermethylation in methylation analysis to have lower expression in THI samples than controls. 
			Similarly, any hypomethylated genes are expected to have higher expression in THI samples.
			If we see expression changes in regions that are not differentially methylated, the implicated region could possibly be affected by a different epigenetic mechanism such as a chromatin modification.
			Any changes that aren't correlated with methylation changes should be further investigated as potential causes of THI.
			

			
	\section{Conclusion}
	
		It is feasible that THI is caused by a delay in maturation as a result of aberrant methylation. 
		Through the use of FACS, WGBS and RNA-Seq, we will test if methylation is causing a block in B lymphopoiesis. 
		Should the results show that methylation is not involved in THI, the FACS and expression data will provide avenues for further research.
		
	\appendix
	\section{Appendices}
	
	\subsection{Sample information}

%					\begin{landscape} % Sample table
%						\footnotesize
%						\begin{longtable}[c]{|l | l |p{4cm}|l|l|l|l|l|}
%							\caption{Clinical details of patients with THI. Abbreviations: m, months; y, years; CVI, common variable immunodeficiency; -, unknown; def, d.} \\ \hline 	
%							Individual & Tissue & Age at test  & Gender  & Ethnicity   & Breastfeeding status & Mode of birth & Family history of PID    \\ \hline \hline
%							\endfirsthead
%							\hline
%							Individual & Tissue & Age at test  & Gender  & Ethnicity   & Breastfeeding status & Mode of birth & Family history of PID    \\ \hline \hline
%							\endhead
%							\hline 	\endfoot 
%							\label{table:samples}
%							1  & Whole Blood & 7m, 11m, 1y 7m, 2y 1m                                    & Female & Caucasian & Breastfed  & Vaginal           & None                 \\
%							& Tonsils     & 2y 1m                                                    &        &           &            &                   &  \\ \hline
%							2  & Whole Blood & 8m, 11m, 1y 3m, 1y 6m, 1y 8m, 2y 1m, 2y 4m, 2y 6m, 2y 8m & Male   & Caucasian & Breastfed  & Vaginal           & None                 \\
%							& Tonsils     & 2y 6m                                                    &        &           &            &                   &  \\ \hline
%							3  & Whole Blood & 9m, 1y 3m, 1y 9m, 2y 6m, 2y 9m, 3y 2m, 3y 6m             & Male   & Jewish    & Bottle fed & Caesarian section & None                 \\
%							& Tonsils     & 3y 2m                                                    &        &           &            &                   &  \\ \hline
%							4  & Whole Blood & 10m, 1y 1m, 1y 8m                                        & Male   & Caucasian & Bottle fed & Vaginal           & None                 \\ \hline
%							5  & Whole Blood & 9m, 11m, 1y 3m, 1y 5m, 1y 8m                             & Male   & Asian     & Breastfed  & Vaginal           & Brother with CVI     \\ \hline
%							6  & Whole Blood & 7m, 9m, 11m, 1y 8m                                       & Female & Jewish    & Breastfed  & Vaginal           & None                 \\ \hline
%							7  & Whole Blood & 9m, 11m, 1y 2m, 1y 6m                                    & Male   & -         & Breastfed  & -                 & -                    \\ \hline
%							8  & Whole Blood & 7m, 9m,                                                  & Female & -         & Breastfed  & -                 & -                    \\ \hline
%							9  & Whole Blood & 7m, 9m, 11m, 1y 3m 1y 8m                                 & Male   & -         & Breastfed  & -                 & -                    \\ \hline
%							10 & Whole Blood & 7m, 9m, 11m, 1y 2m, 1y 6m                                & Male   & -         & Breastfed  & -                 & -                    \\ \hline
%							11 & Whole Blood & 9m, 11m, 1y 1m, 1y 3m 1y 8m                              & Male   & -         & Bottle fed & -                 & -                    \\ \hline
%							12 & Whole Blood & 7m, 11m, 1y 2m, 1y 11m, 2y 6m, 2y 9m, 3y 4m, 3y 7m, 4y   & Female & Jewish    & Breastfed  & Vaginal           & None                 \\
%							& Tonsils     & 2y 11m                                                   &        &           &            &                   &  \\ \hline
%							13 & Whole Blood & 9m, 11m, 1y 2m, 1y 6m, 2y 4m, 2y 8m                      & Male   & Jewish    & Bottle fed & Vaginal           & None                 \\ \hline
%							14 & Whole Blood & 9m, 1y 1m, 1y 7m, 2y 2m, 2y 9m, 3y 2m                    & Female & Caucasian & Bottle fed & Vaginal           & None                 \\
%							& Tonsils     & 3y 2m                                                    &        &           &            &                   &  \\ \hline
%							15 & Whole Blood & 7m, 11m, 1y 1m                                           & Male   & Caucasian & Breastfed  &                   & Sister with IgA def. \\ \hline
%							16 & Whole Blood & 8m, 1y 1m, 1y 4m, 1y 6m                                  & Male   & -         & -          & Vaginal           & -                    \\ \hline
%							17 & Whole Blood & 9m, 1y 2m, 1y 6m                                         & Male   & -         & -          & Vaginal           & -                    \\ \hline
%							18 & Whole Blood & 7m, 9m, 1y 1m                                            & Female & -         & -          & Caesarian section & -                    \\ \hline
%							19 & Whole Blood & 7m, 9m, 11m, 1y 2m, 1y 6m                                & Male   & -         & -          & Vaginal           & -                    \\ \hline
%							20 & Whole Blood & 9m, 11m, 1y 2m, 1y 6m                                    & Male   & -         & -          & Vaginal           & -                    \\ \hline
%							21 & Whole Blood & 11m, 1y 3m, 1y 5m, 1y 8m                                 & Male   & -         & -          & Caesarian section & -                    \\ \hline
%							22 & Whole Blood & 7m, 9m, 11m, 1y 8m                                       & Female & -         & -          & -                 & None                 \\ \hline
%							23 & Whole Blood & 9m, 1y 2m, 1y 6m                                         & Male   & -         & -          & -                 & None                 \\ \hline
%							24 & Whole Blood & 7m, 9m, 1y 1m, 1y 6m, 1y 11m, 2y 1m                      & Female & -         & -          & -                 & None                 \\
%							& Tonsils     & 2y 4m                                                    &        &           &            &                   &  \\ \hline
%							25 & Whole Blood & 7m, 11m, 1y 3m, 1y 5m, 1y 8m                             & Male   & -         & -          & -                 & Sister with IgA def. \\ \hline
%							26 & Whole Blood & 8m, 1y 1m,                                               & Male   & -         & -          & -                 & None
%						\end{longtable}
%					\end{landscape}	
%\include{samplesTable.tex}
				
	\bibliographystyle{BrittSuperScript}
	\bibliography{../literature}
	
%	\includepdf[pages=-]{../Reading/Transient_hypogammaglobulinemia_of_infancy_Ovadia_and_Dalal_2014.pdf}
%	\includepdf[pages=-]{../Reading/Whole-genome-fingerprint-of-the–DNA-methylome-during-human-B-cell-differentiation.pdf}
	
	
	
\end{document}